### ISSUE TITLE
[Enhancement]  Kyverno Auto Cert Renewal Process Docs

### ISSUE THREAD

**Issue by jamesjose03** [kyverno/website#790]
### Description

Hi, I had recently tried verifying the Auto Cert Renewal Process. but was not able to do so after changing the time in the node to which the kyverno pod was attached. But it didn't get renewed. Could you please add documentation to verify the process? Or how the auto renewal works? 

After going through the code, saw a variable named, timeReserveBeforeCertificateExpiration which is 6 months before the expiration date. Is the certificate renewal triggered 6 months before the expiration date?

It would be helpful if the docs were provided for the same.

**Comment by chipzoller** [comment]
cc @eddycharly

**Comment by eddycharly** [comment]
Couple of infos:
- CA certs validaty is one year.
- TLS certs validity is 150 days.
- Periodic certs checks happen every 12 hours
- Certs checks also happen when events are received for managed secrets containing the certs (or when controller resyncs)

Renewal process:
1. remove expired certs
2. check if remaining certs will become invalid in less than 60 hours
3. if needed we generate a new certificate with the validity documented above
4. the new cert is added to the underlying secret along with current certs that are valid

Basically, certs will be renewed 60 hours before expiry.

**Comment by chipzoller** [comment]
Please add in the appropriate section.
